src/backend/mailbox_manager/api/serializers.py

```diff
@@ -16,16 +16,29 @@ class Meta:
 class MailDomainSerializer(serializers.ModelSerializer):
     """Serialize mail domain."""
 
+    slug = serializers.SerializerMethodField()
+
     class Meta:
         model = models.MailDomain
         lookup_field = "slug"
         fields = [
             "id",
             "name",
+            "slug",
+            "created_at",
+            "updated_at",
+        ]
+        read_only_fields = [
+            "id",
+            "slug",
             "created_at",
             "updated_at",
         ]
 
+    def get_slug(self, instance):
+        """Return slug from the domain's name."""
+        return instance.get_slug()
+
```

Comment:
Why do you need to override `slug` ? 
When you save your slug, you already use slugify I can see.
https://github.com/numerique-gouv/people/blob/dc53ecc3bc2ca5d11451ecff8af035fdf3f85230/src/backend/mailbox_manager/models.py#L32-L33

Reply:
Actually you're right, I don't think we need that. I removed it.